sops.go

```diff
@@ -58,6 +59,11 @@ import (
 // DefaultUnencryptedSuffix is the default suffix a TreeItem key has to end with for sops to leave its Value unencrypted
 const DefaultUnencryptedSuffix = "_unencrypted"
 
+var (
+	MasterKeyTypes             = []string{"pgp", "gcp_kms", "hc_vault", "kms", "azure_kv", "age"}
+	SopsDecryptionOrderDefault = []string{"age", "pgp"}
+)
```

Comment:
Am _really_ on the fence about this implementation detail. The problem I have with this introduction is that it worsens the scattering of string identifiers throughout the code base, and can easily be forgotten to be updated when a new key source is introduced.

The first solution to mitigate this that comes to mind would be to start making use of the identifier constants exposed in the key source implementation (as I suggested above), but this would cause the abstraction to further leak details as we would now have to import every key implementation within this module. Which isn't a great solution either.

Which makes me think we should either:

1. Factor this out (but I have no suggestion on how to do this yet...).
2. As a short-term solution, not maintain a list of _all_ key types, but rather only sort based on what's provided (which means we can either rely on our own defaults with a limited set of identifiers, or rely on the user their input).

Reply:
@hiddeco I agree, this list is a bit problematic. I've updated the sort to rely only on provided list.